beaker/build_options.py

```diff
@@ -24,6 +24,15 @@ class BuildOptions:
         the compiled program's size. Enabling this option requires a minimum AVM version of 3.
         Defaults to True."""
 
+    with_sourcemaps: bool = False
+    """ """
```

Comment:
nit: still on going comments?

Reply:
ah yea will update this with cp/pasta of pyteal docstrings